src/main/java/BasicJavaHomeWork/Exercise_2.java

```diff
@@ -0,0 +1,25 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_2 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int num = in.nextInt();
+        int i = num % 3;
+        int a = num % 5;
+
+        if (a == 0 && i == 0){
+            System.out.println(num + " divided by 3 and 5");
+        } else if (i == 0){
+            System.out.println(num + " divided by 3");
+        } else if (a == 0) {
+            System.out.println(num + " divided by 5");
+        } else {
+            System.out.println(num + " NOT divided by 3 and 5");
```

Comment:
please change the messages from "divided" to "is divisible" and "not divided" to "is not divisible"

Reply:
Thank you, I've updated.